### PR TITLE
help page update done:

### PR DESCRIPTION
**Pull request by sheenaze**
- some screens updated to the current versions,
- some text added,
- networks fixture updated.

### CODE REVIEW

File: validator/templates/validator/help.html [hunk 1]
```diff
@@ -71,7 +71,21 @@ <h1 class="h3 mb-3 font-weight-normal">Validate Page</h1>
     <p>
         <b>Step 5 [optional]</b> - Choose the criteria by which you would like to filter the reference data prior to
         running the validation. The filters available depend on the data contained within the chosen dataset. For example,
-        you can filter the ISMN data to include only data points where the soil_moisture_flag is &quot;G&quot; for &quot;good&quot;.
+        you can filter the ISMN data to include only data points where the soil_moisture_flag is &quot;G&quot; for &quot;good&quot;. You can also choose particular
+        networks and depth range at which the measurements have been done. To do that you have to open an appropriate window by clicking a 'select...'
```
Comment (pstradio) [inline validator/templates/validator/help.html:75]: Could you use: '...also choose specific networks and/or measurements in a defined depth range'

File: validator/templates/validator/help.html [hunk 2]
```diff
@@ -71,7 +71,21 @@ <h1 class="h3 mb-3 font-weight-normal">Validate Page</h1>
...
+        link next to the filter you want to parameterised.
+    </p>
+    <img class="scaled_image mb-3" src="../../static/images/help/networks.png">
+    <p>
+        The window for choosing networks contains a list of continents. When clicking a continent name a list of networks that belong to the continent folds down.
```
Comment (pstradio) [inline validator/templates/validator/help.html:80]: folds down -> unfolds

File: validator/templates/validator/help.html [hunk 3]
```diff
@@ -71,7 +71,21 @@ <h1 class="h3 mb-3 font-weight-normal">Validate Page</h1>
...
+        Next to each network there is a name of the country that the network belongs to given. To toggle the particular network it is enough to click on its name.
```
Comment (pstradio) [inline validator/templates/validator/help.html:81]: 'Next to each network there is the name of the country it belongs to'

File: validator/templates/validator/help.html [hunk 4]
```diff
@@ -71,7 +71,21 @@ <h1 class="h3 mb-3 font-weight-normal">Validate Page</h1>
...
+        The choice has to be approved clicking the 'Use networks' button.
```
Comment (pstradio) [inline validator/templates/validator/help.html:82]: 'by' clicking

File: validator/templates/validator/help.html [hunk 5]
```diff
@@ -71,7 +71,21 @@ <h1 class="h3 mb-3 font-weight-normal">Validate Page</h1>
...
+    </p>
+    <img class="scaled_image mb-3" src="../../static/images/help/depths.png">
+    <p>
+        The window for the depth selection contains two fields for introducing the depth range. Again, the choice has to be accepted clicking the 'Use depth range' button.
```
Comment (pstradio) [inline validator/templates/validator/help.html:86]: 'by' clicking :)

File: validator/templates/validator/help.html [hunk 6]
```diff
@@ -98,9 +112,10 @@ <h1 class="h3 mb-3 font-weight-normal">Validate Page</h1>
     <img class="scaled_image mb-3" src="../../static/images/help/map_selection.png">
     <p>
         <b>Step 8 [optional]</b> - Choose the date range over which the validation should be performed. Accepted formats are: YYYY*MM*DD
-        or DD*MM*YYYY where * can be any of &quot;.&quot;, &quot;/&quot; or &quot;-&quot;. If you don't provide a period it will be
-        implicitly determined through temporal matching of the data and reference selected. For the time range covered by the various
-        datasets, see the <a href="{% url 'datasets' %}">datasets page</a>.
+        or DD*MM*YYYY where * can be any of &quot;.&quot;, &quot;/&quot; or &quot;-&quot;. It is also possible to choose a date from a calendar, available under the calendar button.
+        By default, the date range is determined through temporal matching of the data and reference selected.
```
Comment (pstradio) [inline validator/templates/validator/help.html:116]: '...of the selected data and reference'

File: validator/templates/validator/help.html [hunk 7]
```diff
@@ -196,7 +213,8 @@ <h1 id="help3" class="h3 mb-3 font-weight-normal">My Validations - Results List
             </li>
             <li>
                 <span class="fas fa-times"></span> The remove button is only visible once the validation has finished
-                or was aborted and will permanently delete the result.
+                or was aborted and will permanently delete the result. The remove button next to a validation being tracked,
```
Comment (pstradio) [inline validator/templates/validator/help.html:216]: I think there shouldn't be a comma after 'tracked'

File: validator/templates/validator/help.html [hunk 8]
```diff
@@ -49,7 +49,7 @@ <h1 class="h3 mb-3 font-weight-normal">Validate Page</h1>
     <p>
```
Comment (pstradio) [inline validator/templates/validator/help.html:49]: I know it was there before, but could you change the first sendetnce: 'Each area has menu with two icons on the top right.' to 'Each area has a menu with two icons on the top right.'. Also, what is the area exactly?

**Review by pstradio — COMMENTED** [review]
Hey @sheenaze, I left some comment on the text; I also opened the page locally and everything looks good, thanks for the update

**Comment by sheenaze** [comment]
Thanks for the remarks! I'll take a look at them and introduce appropriate changes :)

File: validator/templates/validator/help.html (hunk 8 above)
Comment (sheenaze) [reply validator/templates/validator/help.html:49]: I think the area refers to each setting component. It wasn't me who was writing it :)

**Comment by sheenaze** [comment]
Done

**Pull request merged**